runtime/rasdump/dmpsup.c

```diff
@@ -559,6 +559,7 @@ configureDumpAgents(J9JavaVM *vm)
 	 */
 
 	xdumpIndex = FIND_ARG_IN_VMARGS_FORWARD(OPTIONAL_LIST_MATCH, VMOPT_XDUMP, NULL);
+	char *optionStringOriginal = NULL;
```

Comment:
This could be named better.

Reply:
@pshipton Would `optionStringCopy` work a bit better? I'm going for something that captures that it is meant to keep track of the original pointer.

Reply:
The mechanism doesn't work, as all mapped options need to be freed and not just the last one. But optionStringAllocated seems a better name.

Reply:
Will update name to `optionStringAllocated` and work on a mechanism that frees each mapped option.

Reply:
Note, if you go with what Keith suggested, you won't need this variable any more.